tests/utils.go

```diff
@@ -469,6 +469,64 @@ func GetSupportedCPUModels(nodes k8sv1.NodeList) []string {
 	return cpus
 }
 
+func GetSupportedCPUFeaturesAmongAllTheNodes(nodes k8sv1.NodeList) []string {
```

Comment:
IMO, doesn't make sense to name this function `GetSupportedCPUFeaturesAmongAllTheNodes` and pass it a list of nodes. What if you pass it a subset of nodes ? 

Either make the func read the full list of nodes, or name the method `GetSupportedCPUFeaturesFromNodes` - or something similar. 

My preference is clearly the later option (renaming the func).

Reply:
Hey Thanks for reviewing.
Done.